test/test.js

```diff
@@ -48,14 +48,23 @@ describe('details-menu element', function() {
       assert.equal(summary, document.activeElement, 'escape focuses summary')
     })
 
-    it('updates the button label', function() {
+    it('updates the button label with text', function() {
       const details = document.querySelector('details')
       const summary = details.querySelector('summary')
       const item = details.querySelector('button')
       assert.equal(summary.textContent, 'Click')
```

Comment:
Maybe throw in an HTML tag here to verify the contents is set as text rather than html.